app/javascript/packages/address-search/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@18f/identity-address-search",
-  "version": "2.2.0",
+  "version": "2.3.0",
   "type": "module",
```

Comment:
This should not be a breaking change, to my understanding so I went with a minor update (backwards compatable).  
See file app/javascript/packages/address-search/types.d.ts

noInPersonLocationsDisplay is optional and if a nothing is passed in, will use a default.

```
 noInPersonLocationsDisplay?: ComponentType<{ address: string }>;
 ```